pkg/controllers/provisioning/scheduling/topology.go

```diff
@@ -323,7 +345,17 @@ func (t *Topology) countDomains(ctx context.Context, tg *TopologyGroup) error {
 func (t *Topology) newForTopologies(p *corev1.Pod) []*TopologyGroup {
 	var topologyGroups []*TopologyGroup
 	for _, cs := range p.Spec.TopologySpreadConstraints {
```

Comment:
```suggestion
	for _, tsc := range p.Spec.TopologySpreadConstraints {
```

this is also minor but why not?